### DISCUSSION TITLE
PIMCORE  as CRM

### DISCUSSION THREAD

**Discussion by ismetsener** [pimcore/pimcore#9336] [Ideas]
Hello,

Does any one have experience with Pimcore as CRM
when I look at CRM systems:
customer database
customer activites, like phone calls, tasks, cases, contact requests etc 
### **this should be possible with Pimcore :)** 
in this case you do not need extra system for CRM

**Comment by passioneight** [comment]
You might want to have a look at the [Customer Management Framework](https://pimcore.com/docs/customer-management-framework/current/).

Beware that you'll still have to configure and implement stuff yourself, as this is only a framework.

**Reply by ismetsener** [reply]
I know the  "Customer Management Framework"  
this is just a good basis, it shows what is possible in Pimcore

but I was wondering if someone already build on this to make a "operational"  CRM
I think  that there will be a lot of  potential customers interested in this CRM-build/addon

If you look at the most opensource CRM  packages, then PIMCORE + CRM will outperform them

**Reply by passioneight** [reply]
Not sure if there's any bundle for this yet. Maybe the guys at [Dachom Digital](https://github.com/dachcom-digital) have something like this open source. 

In any case, implementing the code based on the CMF is probably the best option, as you'll get tailored functionality on project-level.

**Reply by ismetsener** [reply]
I will look at their site: https://github.com/dachcom-digital
I will talk in near future with some PHP/Symfony developers, and explain what I want.
I agree with you:
In any case, implementing the code based on the CMF is probably the best option, as you'll get tailored functionality on project-level.
